spec/models/conference_spec.rb

```diff
@@ -7,6 +7,22 @@
   it { is_expected.to have_many(:attendees) }
   it { is_expected.to validate_presence_of(:name) }
   it { is_expected.to validate_presence_of(:round) }
+  it { is_expected.to validate_presence_of(:starts_on) }
+  it { is_expected.to validate_presence_of(:ends_on) }
+
+  describe 'it checks the chronological order of dates' do
+    subject { FactoryGirl.build(:conference) }
```

Comment:
```ruby
expect(subject).not_to be_valid
```
would generally be fine, if you can make sure that the error is not triggered from a completely different validation. So if doing it like this maybe also quickly check if the subject otherwise is valid

```ruby
subject { thing } 

it { is_expected.to be_valid }

# more specs
```